tmcprototype/SubarrayNode/test/SubarrayNode_test.py

```diff
@@ -109,45 +115,65 @@ def test_AssignResources(self, tango_context):
-        """Test for Configure"""
-        # PROTECTED REGION ID(SubarrayNode.test_Configure) ENABLED START #
-        tango_context.device.Configure(['{"scanID":12345,"pointing":{"target":{"system":"ICRS","name":"NGC6251","RA":"2:31:50.91","dec":"89:15:51.4"}},"dish":{"receiverBand":"1"},"csp":{"frequencyBand":"1","delayModelSubscriptionPoint":"","visDestinationAddressSubscriptionPoint":"","fsp":[{"fspID":"1","functionMode":"CORR","frequencySliceID":1,"integrationTime":1400,"corrBandwidth":0,"channelAveragingMap":[]},{"fspID":"2","functionMode":"CORR","frequencySliceID":1,"integrationTime":1400,"corrBandwidth":0,"channelAveragingMap":[]}]},"sdp":{"configure":{"id":"realtime-20190627-0001","sbiId":"20190627-0001","workflow":{"id":"vis_ingest","type":"realtime","version":"0.1.0"},"parameters":{"numStations":4,"numChanels":372,"numPolarisations":4,"freqStartHz":0.35e9,"freqEndHz":1.05e9,"fields":{"0":{"system":"ICRS","name":"NGC6251","ra":"2:31:50.91","dec":"89:15:51.4"}}},"scanParameters":{"12345":{"fieldId":0,"intervalMs":1400}}},"configureScan":{"scanParameters":{"12346":{"fieldId":0,"intervalMs":2800}}}}}'])
-        time.sleep(60)
-        print("Subarray obsState:",tango_context.device.obsState)
-        assert tango_context.device.obsState == 2
-        time.sleep(45)
-        create_dish_proxy.SetPointingState()
-        # PROTECTED REGION END #    //  SubarrayNode.test_Configure
-
-    def test_Scan(self, tango_context):
-        """Test for Scan"""
-        # PROTECTED REGION ID(SubarrayNode.test_Scan) ENABLED START #
-        # with pytest.raises(tango.DevFailed):
-        #     tango_context.device.Scan("a")
-        # assert tango_context.device.obsState != 3
-        tango_context.device.Scan(['{"scanDuration": 10.0}'])
-        time.sleep(5)
-        print ("Scan :tango_context.device.obsState :", tango_context.device.obsState )
-        assert tango_context.device.obsState == 3
-        # with pytest.raises(tango.DevFailed):
-        #     tango_context.device.Scan("0")
-        # assert CONST.SCAN_ALREADY_IN_PROGRESS in tango_context.device.activityMessage
-        # PROTECTED REGION END #    //  SubarrayNode.test_Scan
-
-
-    def test_EndScan(self, tango_context):
-        """Test for EndScan"""
-        # PROTECTED REGION ID(SubarrayNode.test_EndScan) ENABLED START #
-        print("EndScan :tango_context.device.obsState :", tango_context.device.obsState)
-        tango_context.device.EndScan()
-        time.sleep(10)
-        print("EndScan :tango_context.device.obsState :", tango_context.device.obsState )
-        assert tango_context.device.obsState == 2
-        # with pytest.raises(tango.DevFailed):
-        #     tango_context.device.EndScan()
-        # assert CONST.ERR_DUPLICATE_END_SCAN_CMD in tango_context.device.activityMessage
-        # PROTECTED REGION END #    //  SubarrayNode.test_EndScan
+    # def test_Configure(self, tango_context, create_dish_proxy):
+    #     """Test for Configure"""
```

Comment:
Add TODO/FIXME tags for the commented test cases.

Reply:
Done.